opmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -1,10 +1,19 @@
+"""
```

Comment:
We usually start with "This file is part of the openPMD viewer."

For instance here:
```
"""
This file is part of the openPMD viewer.

It contains diagnostics relevant for laser-plasma acceleration.
```